### DISCUSSION TITLE
Regarding uploading proof of academic status

### DISCUSSION THREAD

**Discussion by Chanukaisuru** [community/community#141965] [GitHub Education]
### Hi everyone,

it asks why I am not in campus but I am in campus. It asks the reason for being away from the campus. I need help with this

![Screenshot 2024-10-19 110234](https://github.com/user-attachments/assets/53b25ff4-28bb-4c89-909b-4fb2c24b38a6)

**Comment by davevad93** [comment]
Hi @Chanukaisuru ,

Given that you are submitting the application from your campus, it's possible that GitHub may have incorrect information about your school's location.

Try contacting support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
